stdlib/runtime/string.gr

```diff
@@ -512,13 +513,12 @@ let rec heapValueToString = (ptr, extraIndents, toplevel) => {
         WasmI32.store(ptr, 0x80000000n | tupleLength, 4n)
         let comspace = ", "
         let rparen = ")"
+        let mut lparen = "("
         let mut strings = [rparen]
         if (tupleLength <= 1n) {
           // Special case: unary tuple, which is not valid Grain syntax; however, boxed values
           // are stored as a unary tuple, so we keep this in case one gets printed
-          let comma = ","
-          strings = [comma, ...strings]
-          void
+          lparen = "Box("
```

Comment:
It'd be a bit better if in this case you appended `"box"` to the list that's all going to get concatenated, so just `strings = ["box", ...strings]`. You might need to rework things a bit and push this `if` til after the loop. We'll also want to do lowercase `"box"` 🙂

Reply:
Done this change.